dash-server.mjs

```diff
@@ -15,7 +15,7 @@ import { readFile } from 'fs/promises';
 var time=0;
 
 // Load initial data
-let data = JSON.parse(await readFile("devices.json", "utf8"));
+let data = JSON.parse(await readFile("devices-example.json", "utf8"));

```

Comment:
The idea here is like in Laravel where you never check in your actual file with the sensitive info. There should just be an example file with the correct structure, but it would be copied over to the environment where the actual printers are. So this change on line 18 should be reverted. Also, the passwords in the devices-example.json should not be checked in to source control, but in your personal devices.json file, which is excluded by .gitignore.